src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2546,7 +2546,15 @@ internal void RequestValueUpdate(EngineController engine)
             };
 
             var task = new UpdateRenderPackageAsyncTask(scheduler);
-            if (!task.Initialize(initParams)) return false;
+            try
+            {
+                if (!task.Initialize(initParams)) return false;
```

Comment:
Can't we simply add a null check for `initParams` if that's what's going on here?